cmd/erc20.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+
+	"github.com/datachainlab/ethereum-ics20-cli/chains/geth"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/client"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/erc20"
+	"github.com/spf13/cobra"
+)
+
+func erc20Cmd() *cobra.Command {
+	var rpcAddress, mnemonic, ics20BankAddress, ics20TransferBankAddress string
+	var fromIndex uint32
+	var toAddress string
+	var amount int64
+	var denom string
+	var walletAddress string
+
+	cmd := &cobra.Command{
+		Use:   "erc20",
+		Short: "erc20 command",
+	}
+
+	// balance command
+	balanceCmd := &cobra.Command{
+		Use:   "balance",
+		Short: "balance of the account",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			balance, err := erc20Balance(rpcAddress, walletAddress, denom)
+			if err != nil {
+				return err
+			}
+			fmt.Printf("%d\n", balance)
+			return nil
+		},
+	}
+	balanceCmd.Flags().StringVar(&rpcAddress, "rpc-address", "", "Ethereum RPC Address")
+	balanceCmd.Flags().StringVar(&walletAddress, "wallet-address", "", "Wallet address")
+	balanceCmd.Flags().StringVar(&denom, "denom", "", "Token denom")
+
+	balanceCmd.MarkFlagRequired("rpc-address")
+	balanceCmd.MarkFlagRequired("wallet-address")
+	balanceCmd.MarkFlagRequired("denom")
```

Comment:
I think you should put each subcommand's definition on a different file.
One reason of this is that, for example, the logic of `balanceCmd` can refer to `fromIndex` or `toAddress` undesirably.

Reply:
Changed file structure
```
├── cmd
│   ├── chain.go
│   ├── erc20
│   │   ├── balance.go
│   │   ├── erc20.go
│   │   └── transfer.go
│   ├── ics20
│   │   ├── balance.go
│   │   ├── ics20.go
│   │   └── transfer.go
│   ├── root.go
│   └── wallet.go
```